board/opentrons/ot2/rootfs-overlay/usr/bin/ot-status-leds

```diff
@@ -269,14 +269,17 @@ async def main():
     """ If we have a refresh board, initialize LEDs
         and run status update tasks
     """
+    refresh_board = is_refresh_board()
+
+    """Systemd can start next service once revision pins have been read. """
```

Comment:
Should this be

```suggestion
    # Systemd can start next service once revision pins have been read.
```

Reply:
Also, I'd personally find it helpful if this explained *why* we need to notify systemd before it starts the next service. (What is the next service? Why is it important that it doesn't run before or during us reading the revision pins?)

Reply:
Need for notifying and sequence of services is explained in PR #116 and its comments.

Reply:
Relevant comment thread: https://github.com/Opentrons/buildroot/pull/116#pullrequestreview-554845415